packages/components/src/session.tsx

```diff
@@ -1,137 +0,0 @@
-import { Signer } from '@ethersproject/abstract-signer'
-import {
-  ISessionContext,
-  SessionContext,
-  useAuthenticate,
-  useEagerConnect,
-  useSigner,
-} from '@nft/hooks'
-import { useWeb3React } from '@web3-react/core'
-import decode, { JwtPayload } from 'jwt-decode'
-import React, { FC, useCallback, useEffect, useMemo } from 'react'
-import { useCookies } from 'react-cookie'
-
-export const COOKIE_ADDRESS = 'address'
-export const COOKIE_JWT_TOKEN = (address: string): string =>
-  `jwtToken-${address.toLowerCase()}`
-
-const cookieOptions = {
-  secure: true,
-  sameSite: true,
-  path: '/',
-}
-
-const currentJWT = (cookies: {
-  [key: string]: string
-}): null | { address: string; jwt: string } => {
-  const account = cookies[COOKIE_ADDRESS]
-  if (!account) return null
-  const jwtToken = cookies[COOKIE_JWT_TOKEN(account)]
-  if (!jwtToken) return null
-  const res = decode<JwtPayload & { address: string }>(jwtToken) // TODO: use `sub` instead of `address`?
-  if (res.exp && res.exp < Math.ceil(Date.now() / 1000)) return null
-  return { address: res.address.toLowerCase(), jwt: jwtToken }
-}
-
-export const currentAccount = (cookies: {
-  [key: string]: string
-}): string | null => {
-  const res = currentJWT(cookies)
-  if (!res) return null
-  return res.address
-}
-
-export type Props = {
-  connectors: ISessionContext['connectors']
-}
-
-const Session: FC<Props> = ({ connectors, ...props }) => {
```

Comment:
Authentication delegated to the application in the `AccountProvider` within the app.
It is reimplementing a similar logic with a single jwt token in the cookies instead of one per address to keep things much simpler